jenkins-integration/beaker/gatling/watch_puppetserver_mem.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
I grabbed this script off of one of the SUTs and cleaned it up quite a bit. But it's probably got room for improvement. 

Also not sure if this is a good place to store the script